### ISSUE TITLE
[Repo]: Constants no longer used for number strings in test files

### ISSUE THREAD

**Issue by YakDriver** [hashicorp/terraform-provider-aws#39881]
### TL;DR

**We apologize for any inconvenience if your PR was broken by changes to constants in test files!**

What happened:

1. Number-string (_e.g._, `"1"`) constants (_e.g._, `acctest.Ct1`) are no longer defined -- this only affects _test_ files
2. As you might guess, CI/Semgrep checks now don't enforce use of constants for number strings

Fixing your PR:

* If your PR includes changes to test files, replace constants with number-strings, _e.g._, replacing `acctest.Ct1` with `"1"` (see below for autofix instructions)

### More details

Previously, in tests, we used number-string (_e.g._, `"1"`) constants (_e.g._, `acctest.Ct1`) for common numbers:

```go
resource.TestCheckResourceAttr(resourceName, "action.#", acctest.Ct1),
```

As maintainers, we made the decision that this pattern was _not_ a net benefit and made the code harder to read. As a result, we have reverted all number-string constants back to number-strings in test files:

```go
resource.TestCheckResourceAttr(resourceName, "action.#", "1"),
```

#### Process 1

For most PRs, with changes in only a few test files, this is how to autofix your PR:

1. Switch to the `main` branch and pull the latest
2. Rebase your branch on the freshly pulled `origin/main` _OR_ merge `origin/main` onto your branch
3. Fix merge conflicts
4. Run `make fix-constants`

This will replace constants throughout the test codebase with the right number string: _e.g._, `acctest.Ct1` will be replaced with `"1"`.

#### Process 2

For bigger PRs, affecting many test files, this advanced approach can reduce the merge conflicts. In most cases, you should use Process 1.

1. `git checkout ed86924 -- GNUmakefile .ci/.semgrep-test-constants-temp.yml`
2. Run `make fix-constants`
3. Switch to the `main` branch and pull the latest
4. Rebase your branch on the freshly pulled `origin/main` _OR_ merge `origin/main` onto your branch
5. Fix merge conflicts

### References

#39840
